src/components/dashboard/NewTripFormContainer.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import NewTripForm from './NewTripForm';
+import { addTripToDb } from '../../redux/actions/tripActions';
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    close: ownProps.close
```

Comment:
I haven't seen this `ownProps` before. This is a method, though? My understanding is that it closes the modal which a component is contained in?

Reply:
ownProps is just the typical style referring to props passed to a container (other than the state). In this case yupp, the method to close the modal